ledger/src/blockstore.rs

```diff
@@ -944,81 +961,101 @@ impl Blockstore {
     /// Collects and reports [`BlockstoreRocksDbColumnFamilyMetrics`] for the
     /// all the column families.
     pub fn submit_rocksdb_cf_metrics_for_all_cfs(&self) {
+        let advanced_options = self.advanced_options.clone();
```

Comment:
I know this struct is small but could this be a borrow instead of a clone?

Reply:
Let me give it a try!